pkg/base-dev/sur/hood.hoon

```diff
@@ -103,6 +104,18 @@
       leaf/"source aeon:      {?~(rail.arak <~> <aeon.u.rail.arak>)}"
       leaf/"pending updates:  {pen}"
   ==
+::  +report-kids: non-vat cz hash report for kids desk
+::
+++  report-kids
+  |=  [our=ship now=@da]
+  ^-  tank
+  =/  dek  %kids
+  =/  ego  (scot %p our)
+  =/  wen  (scot %da now)
+  ?.  (~(has in .^((set desk) %cd /[ego]/base/[wen])) dek)
```

Comment:
You could consider leaving out the desk here because it doesn't affect the scry.

```suggestion
  ?.  (~(has in .^((set desk) %cd /[ego]//[wen])) dek)
```